package.json

```diff
@@ -26,18 +28,18 @@
     "@typescript-eslint/parser": "^5.47.1",
     "eslint": "^8.31.0",
     "jest": "^29.3.1",
-    "ngrok": "^4.3.3",
+    "ngrok": "^5.0.0-beta.2",
     "pre-commit": "^1.2.2",
     "prettier": "^2.8.1",
     "ts-jest": "^29.0.5",
     "ts-node": "^10.9.1",
     "typescript": "^4.9.4",
-    "wrangler": "^2.5.0"
+    "wrangler": "^3.1.1"
```

Comment:
older versions are getting deprecated

Reply:
Have you checked the release notes and ensured the required changes?

Reply:
Yes the following version works exactly same for the production but developers now will have a little more work to do in terms of setting and saving secrets.

Reply:
Is this been documented?

Reply:
Please have it documented
Rest looks good